Flaky test: org.keycloak.testsuite.broker.KcOidcBrokerTokenExchangeTest#testInternalExternalTokenExchangeStoredToken

## org.keycloak.testsuite.broker.KcOidcBrokerTokenExchangeTest#testInternalExternalTokenExchangeStoredToken
[Keycloak CI - Base IT (2)](https://github.com/keycloak/keycloak/actions/runs/12400941407/job/34619413585) / [Pull Request #36020](https://github.com/keycloak/keycloak/pull/36020)

```
java.lang.AssertionError: 

Expected: a value greater than <5L>
     but: <5L> was equal to <5L>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
...
```

--- a/services/src/main/java/org/keycloak/broker/oidc/OIDCIdentityProvider.java
+++ b/services/src/main/java/org/keycloak/broker/oidc/OIDCIdentityProvider.java
@@ -204,7 +204,7 @@ public class OIDCIdentityProvider extends AbstractOAuth2IdentityProvider<OIDCIde
             AccessTokenResponse tokenResponse = JsonSerialization.readValue(modelTokenString, AccessTokenResponse.class);
             Integer exp = (Integer) tokenResponse.getOtherClaims().get(ACCESS_TOKEN_EXPIRATION);
             final int currentTime = Time.currentTime();
-            if (exp != null && exp < currentTime + getConfig().getMinValidityToken()) {
+            if (exp != null && exp <= currentTime + getConfig().getMinValidityToken()) {
                 if (tokenResponse.getRefreshToken() == null) {
                     return exchangeTokenExpired(uriInfo, authorizedClient, tokenUserSession, tokenSubject);
                 }

